inst/report/KRIReportBySite.Rmd

```diff
@@ -144,6 +149,14 @@ overview_table
 
 :::
 
+```{r, echo=FALSE, results = 'asis', warning=FALSE}
+cat(glue::glue("As of {snapshot_date}, {study_id} has {subjects} participants enrolled across 
+{length(unique(overview_raw_table$Site))} sites. {red_kris} Site-KRI combinations have been flagged as red across {overview_raw_table %>% filter(.data$`Red KRIs` != 0) %>% nrow()} sites as shown in the Study Overview Table above\n
```

Comment:
```suggestion
```{r, echo=FALSE, results = 'asis', warning=FALSE, eval=status_study_exists }
cat(glue::glue("As of {snapshot_date}, {study_id} has {subjects} participants enrolled across 
{length(unique(overview_raw_table$Site))} sites. {red_kris} Site-KRI combinations have been flagged as red across {overview_raw_table %>% filter(.data$`Red KRIs` != 0) %>% nrow()} sites as shown in the Study Overview Table above\n
```

Reply:
This will now conditionally render the text whenever `params$status_study` is passed through to `Study_Report()`